src/static_analysis/mod.rs

```diff
@@ -12,6 +17,10 @@ use self::code::*;
 use results::{Results, Benchmark};
 use Config;
 
+/// Runs the analysis for manifest, certificate and code files.
+///
+/// * Benchmarking support.
+
```

Comment:
I would remove that line, the more close the comments are to the method, the better for easier maintenance.